content/kubelb/main/tutorials/observability/_index.en.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Observability"
+linkTitle = "Observability"
+date = 2023-10-27T10:07:15+02:00
+weight = 7
++++
+
+__WIP__
```

Comment:
Shouldn't we just hide the whole Observability page until it will have content?

Reply:
It's actually in main branch, so the page is always marked as `Work in progress`. I think it's fine to keep it.